cmd/earthly/main.go

```diff
@@ -178,6 +178,13 @@ var (
 	GitSha string
 )
 
+var (
+	errLoginFLagsHaveNoEffect            = errors.New("account login flags have no effect when --auth-token (or the EARTHLY_TOKEN environment variable) is set")
```

Comment:
```suggestion
	errLoginFlagsHaveNoEffect            = errors.New("account login flags have no effect when --auth-token (or the EARTHLY_TOKEN environment variable) is set")
```

Reply:
:facepalm: good catch. I accidentally propagated this error with vim's auto completion.